Doc/library/datetime.rst

```diff
@@ -438,24 +481,26 @@ Other constructors, all class methods:
 
 .. classmethod:: date.fromordinal(ordinal)
 
-   Return the date corresponding to the proleptic Gregorian ordinal, where January
-   1 of year 1 has ordinal 1.  :exc:`ValueError` is raised unless ``1 <= ordinal <=
-   date.max.toordinal()``. For any date *d*, ``date.fromordinal(d.toordinal()) ==
-   d``.
+   Return the date corresponding to the proleptic Gregorian ordinal, where
+   January 1 of year 1 has ordinal 1.
+
+   :exc:`ValueError` is raised unless ``1 <= ordinal <=
+   date.max.toordinal()``. For any date *d*,
+   ``date.fromordinal(d.toordinal()) == d``.
 
 
 .. classmethod:: date.fromisoformat(date_string)
 
-  Return a :class:`date` corresponding to a *date_string* in the format emitted
-  by :meth:`date.isoformat`. Specifically, this function supports strings in
-  the format(s) ``YYYY-MM-DD``.
+   Return a :class:`date` corresponding to a *date_string* given in the format
+   ``YYYY-MM-DD``.
 
-  .. caution::
+      >>> date.fromisoformat('2019-12-04')
+      datetime.date(2019, 12, 4)
 
-    This does not support parsing arbitrary ISO 8601 strings - it is only intended
-    as the inverse operation of :meth:`date.isoformat`.
+   This is the inverse of :meth:`date.isoformat`.  It only supports the format
```

Comment:
Please remove one space after the full stop.

Reply:
Do you mean to convert the two spaces after the period to one space?

I am fine with doing that, but the docs note that it doesn't really matter: https://devguide.python.org/documenting/#use-of-whitespace

Reply:
It really doesn't matter when the html is built, I just notice them while I was reading. Just try to be consistent with the rest of the page.

Reply:
Okay - I think I caught them all in 2815035.